nix/default.nix

```diff
@@ -33,6 +33,14 @@ in
       src = ../.;
     };
 
+    modules = [{
+      # Remove runtime dependency on GCC
+      packages.PaymentServer.components.exes.PaymentServer = {
```

Comment:
This doesn't seem to catch all executables, looking through the [build output](https://app.circleci.com/pipelines/github/PrivateStorageio/PaymentServer/425/workflows/49a4a828-db3c-4152-9172-fd5c2d8f8f39/jobs/444/steps) and searching for "stripping" still looks like all the exes unchanged.